### PR TITLE
Release/2.0.0 rc.2

### PR DESCRIPTION
**Pull request by anxolin**
## Do not Squash 🥒, just normal merge

```
# To get commits used in the description (useful for next releases) 
git log master..HEAD --oneline --decorate=0 | grep Bump
git log master..HEAD --oneline --decorate=0 | grep -v Bump
```

# 2.0.0 rc.2

## Explorer Highlights
- New home page with a search input
- In the order details, if you load one order and doesn't exist for the current network, it'll try to find the order for the other networks and do an automatic redirection
- Remove all fake data, a temporal loader
- Include the link to the allowance manager contract in the footer
- Integrate with Google Analytics
- Add order status, and % of execution progress bar
- Add fee row
- Fix wrongly displayed amounts in order details
- Make app responsive
- Improve styles
- Add tooltips for the fields of the order details

## Safe app Highlights
- Adds a new app for a possible new Gnosis Safe app
- Integrates it with PRaul: Automatic link generation
- Integrates it with Gnosis Safe SDK 

## Details
85049213 Merge branch 'master' into develop
8e0300cd From amount includes fee amount (#289)
30c9d875 Fix filled full amount (#292)
83e90f5f Find order, even if you are not in the right network (#290)
5326575e Integrate Safe app sdk (#278)
99eeaf48 Safe app praul (#277)
9dd0738a Safe app praul (#277)
ed863e3c Add readme for apps (#275)
47bc12fd Add safe-swap app (#270)
dbd2ba0f Home search progress. (#283)
52a08e2e Pr256/comments (#279)
4da5595b Pr244/comments (#280)
3d7f470d Do not display loading indicator when order is already loaded (#281)
55a7ff9f 250/hide mocked data (#254)
9b837dd8 Footer suggestion (#271)
dd2e13bc 251/update contract address (#256)
3e86aa8d Was using wrong amounts to show the filled amount (#257)
bbd2c08e Checksum token images (#258)
9b83ccce Upgrade ts (#253)
0b0633c5 232/filled row logic (#244)
94965c29 Analytics: Add external link support (#242)
c75a14ff Add network dimension (#241)
5f4dffba [Analytics] Add copy event (#239)
70fad081 Add Analytics support (#237)
c6655164 Use a different infura id for explorer (#240)
4d6fb336 Style header/networkLabel/footer. (#236)
508defcd 178/update order statuses (#214)
694da358 Adding fill progress bar. (#235)
28289653 212/gas fee paid row (#233)
04254576 Making the order detail view responsive. (#231)
7a7ea904 207/network state on storybook (#210)
528a9ee2 Fix non-boolean attribute warning for CopyButton (#213)
c023528a Improve order detail styles. (#208)
f3f548ff Use inter-ui font import + variable font & cleanup. (#206)
875ec8a2 Fully automate dev setup with Gitpod (#204)
0a7fffdb Pr205/comments (#209)
038f7d08 Merge pull request #201 from gnosis/gp-tooltips
38679609 Move legacy files
beef43c5 Resolve conflict + repurpose 'amount' tooltip.
f70c9b13 168/amount row (#205)
c9d39570 Move legacy tooltip.
a47aa817 Moved actual content out of CSS and into JSX (#203)
efdd8a97 Minify question icon.
04944e3f Adding and styling tooltips.
66999730 Adding and styling tooltips.

## Dependency Changes
2c9e81cf Bump react-dev-utils from 10.2.1 to 11.0.4 (#288)
3e80444e Bump @storybook/react from 6.1.17 to 6.1.21 (#260)
8406fa8e Bump eslint-plugin-jest from 24.1.5 to 24.1.8 (#259)
2d7172a9 Bump @types/node from 14.14.31 to 14.14.32 (#264)
e00fc654 Bump @types/react-router from 5.1.11 to 5.1.12 (#262)
d76799c1 Bump @babel/preset-env from 7.13.8 to 7.13.9 (#263)
832831b5 Bump @types/react from 16.14.2 to 16.14.5 (#265)
255556b4 Bump @babel/core from 7.13.1 to 7.13.8 (#267)
85b30278 Bump eslint from 7.19.0 to 7.21.0 (#266)
4fc2d6fa Bump @storybook/addon-actions from 6.1.20 to 6.1.21 (#268)
eada6d19 [Security] Bump elliptic from 6.5.3 to 6.5.4 (#276)
c2388008 Bump typescript from 4.1.3 to 4.2.3 (#261)
ae9a23ea Bump bn.js from 4.11.9 to 4.12.0 (#215)
b7e6a840 Bump @babel/preset-env from 7.12.17 to 7.13.8
60f4f78d Bump lightweight-charts from 3.2.0 to 3.3.0 (#218)
70cb4aec Bump ts-jest from 26.5.1 to 26.5.2 (#216)
859bc21a Bump @babel/register from 7.12.13 to 7.13.8 (#220)
6223274f Bump stylelint from 13.9.0 to 13.11.0 (#223)
024d9469 Bump @storybook/addon-essentials from 6.1.18 to 6.1.20 (#217)
172d1f87 Bump playwright from 1.8.0 to 1.9.1 (#219)
92504b88 Bump stylelint-prettier from 1.1.2 to 1.2.0 (#222)
0c2b739a Bump @storybook/addon-links from 6.1.18 to 6.1.20 (#226)

### CODE REVIEW

**Review by W3stside — APPROVED** [review]

File: src/theme/types.ts [hunk 1]
```diff
@@ -47,3 +47,8 @@ declare module 'styled-components' {
     mq: DefaultTheme['mediaQueries']
   }
 }
+
+export enum AnalyticsDimension {
+  NETWORK,
+  BROWSER_TYPE,
+}
```
Comment (alfetopito) [inline src/theme/types.ts:54]: Why is this on theme?

File: src/types/index.ts [hunk 2]
```diff
@@ -62,3 +62,8 @@ export interface Fraction {
   denominator: BN
   numerator: BN
 }
+
+export enum AnalyticsDimension {
+  NETWORK,
+  BROWSER_TYPE,
+}
```
Comment (alfetopito) [inline src/types/index.ts:69]: And why is this on root types?

I mean, looks like duplication. Either on theme or here?

**Review by alfetopito — APPROVED** [review]
Only one minor comment, not to be addressed in this PR

File: src/theme/types.ts (hunk 1 above)
Comment (anxolin) [reply src/theme/types.ts:54]: Good catch 🙈

Comment (anxolin) [reply src/theme/types.ts:54]: https://github.com/gnosis/gp-ui/pull/322

File: src/types/index.ts (hunk 2 above)
Comment (anxolin) [reply src/types/index.ts:69]: This is a generic enum with a few metrics that apps use. 
Each app define what's the dimension name in the context of the app. However, you can see how theres some generic components like `src/api/analytics/index.tsx` (initAnalytics). They don't know the name of the dimension, but they can tell if the app has specified a BROWSER_TYPE dimension or not.

**Pull request merged**